lib/pages/new_private_chat/widget/expansion_contact_list_tile.dart

```diff
@@ -26,103 +29,126 @@ class ExpansionContactListTile extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Padding(
-      padding: const EdgeInsets.only(left: 8.0, top: 8.0, bottom: 12.0),
-      child: FutureBuilder<Profile?>(
-        key: contact.matrixId != null ? Key(contact.matrixId!) : null,
-        future:
-            contact.status == ContactStatus.active ? getProfile(context) : null,
-        builder: (context, snapshot) {
-          return Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: [
-              IgnorePointer(
-                child: Avatar(
-                  mxContent: snapshot.data?.avatarUrl,
-                  name: contact.displayName,
-                  size: 48.0,
+    return Container(
+      decoration: BoxDecoration(
+        border: Border(
+          bottom: BorderSide(
+            color: LinagoraStateLayer(LinagoraSysColors.material().surfaceTint)
+                .opacityLayer3,
```

Comment:
IMO, it is `surface` color ( ref from `chat list item`)
<img width="295" alt="image" src="https://github.com/user-attachments/assets/43b7130c-5198-4139-95e7-44b548973018">

Please create a util class to make us reuse it.

Reply:
This is the color for the seperator between items, its the same as in chat list 
 contact list 
![image](https://github.com/user-attachments/assets/9ad676c6-07f2-422e-a264-b9f4b122fae8)
chat list 
![image](https://github.com/user-attachments/assets/ca7f5ebd-4aee-49de-af7d-36f47c6f4f4a)